ml/backend/ggml/ggml/src/ggml.go

```diff
@@ -90,4 +105,37 @@ var OnceLoad = sync.OnceFunc(func() {
 			visited[abspath] = struct{}{}
 		}
 	}
+
+	var attrs []slog.Attr
+	names := make(map[string]int)
+	for i := range C.ggml_backend_dev_count() {
+		r := C.ggml_backend_dev_backend_reg(C.ggml_backend_dev_get(i))
+
+		func() {
```

Comment:
Is this `func() {` needed?

Reply:
Kind of. It ensures `C.free` is run before continuing. Otherwise it won't be freed until the function returns